sequence_processing_pipeline/util.py

```diff
@@ -35,8 +58,22 @@ def iter_paired_files(files):
                 if r2_exp not in r2_fp:
                     raise ValueError(f"Cannot find '{r2_exp}' in '{r2_fp}'")
 
-                r1_prefix = r1_fp[:r1_fp.find(r1_exp)]
-                r2_prefix = r2_fp[:r2_fp.find(r2_exp)]
+                # replace find w/find so that search for R1 and R2 begin
+                # from the end of the string, not the beginning. This prevents
+                # the code from breaking when filenames include R1 and R2 as
+                # part of their name in addition to representing forward and
+                # reversed reads e.g.:
+                # LS_8_22_2014_R1_SRE_S3_L007_R1_001.trimmed.fastq.gz
+                # LS_8_22_2014_R1_SRE_S3_L007_R2_001.trimmed.fastq.gz
+                # using find(), r1_prefix and r2_prefix will be the following:
+                # r1_prefix will be: LS_8_22_2014
+                # r2_prefix will be: LS_8_22_2014_R1_SRE_S3_L007
+
+                # r1_prefix = r1_fp[:r1_fp.find(r1_exp)]
+                # r2_prefix = r2_fp[:r2_fp.find(r2_exp)]
```

Comment:
rm?

Reply:
As long as it's clear to you why I made the change, I'll remove them. I keep those in to remind us at this time.